app/repeating_element/repeating_element.py

```diff
@@ -0,0 +1,72 @@
+from app.schema.answer import Answer
+import logging
+from copy import deepcopy
+logger = logging.getLogger(__name__)
+
+
+class RepeatingElementException(Exception):
+    pass
+
+
+class RepeatingElement(object):
+    ''' The repeating element manager will create a new element with new ids and q-codes using the
+        repetition number as an identifier (e.g. item.id_repetition number)
+    '''
+
+    def __init__(self, questionnaire_manager):
```

Comment:
I think this class should be constructed with the schema, rather than using the one inside questionnaire manager